packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -6,6 +6,16 @@
                 [$style['c-filterPill--disabled']]: isToggleDisabled
             }]"
         data-test-id="filter-item">
+        <a
+            :class="$style['c-filterPill-link']"
+            :href="href"
+            tabindex="-1"
+            aria-hidden="true"
```

Comment:
Do we want this hidden if there's no-js on the page? That's when this should essentially be behaving like a link, I believe.

Might be worth doubling checking if @AlixRichards knows how this is handled in the image tile :)

Reply:
I created a ticket to address this https://justeattakeaway.atlassian.net/browse/SEARCH-2233. Because currently, we don't display anything inside filter pill when js is disabled, I don't think users should be able to click the links. I think we'll have to move this ticket to the next phase though.